app/assets/javascripts/code/services/_sessions_utils.js

```diff
@@ -0,0 +1,96 @@
+import _ from 'underscore';
```

Comment:
This is a new file cause I've decoupled from angular with the usual trick (ie `_`)